docs/lectern-2.0-changes.md

```diff
@@ -21,6 +22,7 @@ The release of Lectern 2.0 brings some important upgrades to the Lectern service
   - Updated interface for Lectern Server REST client
   - Exposes dictionary meta-schema validation, data parsing, and data validation functions
 
+
```

Comment:
probable mistake?
```suggestion
```

Reply:
WIP document... capturing list of changes as bullet points for now so all changes are collected in one place and can be thoroughly described at a later point. Note on `placeholder` content as well.

Reply:
🤷 inconsistent double whitespace. if WIP ignore comment anyway 👍